arsceneview/src/main/kotlin/io/github/sceneview/ar/scene/PlaneRenderer.kt

```diff
@@ -86,14 +86,15 @@ class PlaneRenderer(val lifecycle: ArSceneLifecycle) : ArSceneLifecycleObserver
                 // Do a hittest on the current frame. The result is used to calculate
                 // a focusPoint and to render the top most plane Trackable if
                 // planeRendererMode is set to RENDER_TOP_MOST.
-                val updatedPlanes = arFrame.updatedPlanes
+                val hitResult = getHitResult(arFrame.frame, renderer.desiredWidth, renderer.desiredHeight)
```

Comment:
Here the width and height of the AR view are required. Can we provide them in `ArFrame` or `desiredWidth` and `desiredHeight` are sufficient?